src/v1/device_manager_client.ts

```diff
@@ -910,7 +938,7 @@ export class DeviceManagerClient {
    *   [documentation](https://github.com/googleapis/gax-nodejs/blob/master/client-libraries.md#regular-methods)
    *   for more details and examples.
    * @example <caption>include:samples/generated/v1/device_manager.get_device.js</caption>
-   * region_tag:cloudiot_v1_generated_DeviceManager_GetDevice                                                                                                                     _async
+   * region_tag:cloudiot_v1_generated_DeviceManager_GetDevice_async
```

Comment:
I don't think I've seen a change like this in other PRs. Can you comment as to why this is necessary?

Reply:
Reverted @clarkbynum